include/sway/input/seat.h

```diff
@@ -17,6 +17,7 @@ struct sway_seatop_impl {
 			enum wlr_button_state state);
 	void (*motion)(struct sway_seat *seat, uint32_t time_msec,
 			double dx, double dy);
+	void (*touch_down)(struct sway_seat *seat, uint32_t time_msec);
```

Comment:
Just in terms of placement, this is not a good line to put this on. `button`/`motion`/`axis` all refer to pointers, so having a touch handler in the middle is a bit weird. Same with the decl of the prototype, and `.touch_down = handle_touch_down,`.